docs/content/latest/deploy/manual-deployment/system-config.md

```diff
@@ -115,4 +115,15 @@ After changing a `ulimit` setting in `/etc/security/limits.conf`, you will need
 
 {{< /note >}}
 
+{{< note title="Note" >}}
+If you're using a desktop-distribution, such as ubuntu-desktop, the settings above may not suffice.
+The os needs additional steps to change `ulimit` for gui login. In the case of ubuntu-desktop:
+
+In `/etc/systemd/user.conf` and `/etc/systemd/system.conf`, add at the end of file `DefaultLimitNOFILE=65535`.
```

Comment:
Do we also need to set the max processes setting?

Reply:
And does the recommendation for NOFILE match the number that we suggest in other places?